dev-libs/msgpack/msgpack-2.1.1.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit cmake-multilib
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/${PN}/${PN}-c.git"
+else
+	SRC_URI="https://github.com/${PN}/${PN}-c/releases/download/cpp-${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86 ~x86-fbsd ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="MessagePack is a binary-based efficient data interchange format"
+HOMEPAGE="http://msgpack.org/ https://github.com/msgpack/msgpack-c/"
+
+LICENSE="Boost-1.0"
+SLOT="0"
+IUSE="boost +cxx doc examples static-libs test"
+
+DEPEND="
+	test? (
+		>=dev-cpp/gtest-1.6.0-r2[${MULTILIB_USEDEP}]
+		sys-libs/zlib[${MULTILIB_USEDEP}]
+	)
+	boost? ( dev-libs/boost[static-libs] )
+	doc? ( app-doc/doxygen )
+"
+
+DOCS=( README.md )
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.4.2-cflags.patch
+	"${FILESDIR}"/${PN}-1.4.2-static.patch
+)
+
+src_configure() {
+	local mycmakeargs=(
+		-DMSGPACK_ENABLE_CXX="$(usex cxx)"
+		-DMSGPACK_STATIC="$(usex static-libs)"
+		-DMSGPACK_BUILD_TESTS="$(usex test)"
+		-DMSGPACK_BOOST="$(usex boost)"
+		# Don't build the examples
+		-DMSGPACK_BUILD_EXAMPLES=OFF
```

Comment:
I discussed this on IRC with the devs, they shouldn't be built (I actually did that first), just installed in the /usr/share/doc directory.